packages/menu/src/Menu.ts

```diff
@@ -151,10 +151,9 @@ export class Menu extends SizedMixin(SpectrumElement, { noDefaultSize: true }) {
 
     /**
      * whether or not to support pointerdown - drag - pointerup selection strategy
```

Comment:
```suggestion
/**
 * Whether to support the pointerdown-drag-pointerup selection strategy.
 * Defaults to false to prevent click/touch events from being captured
 * behind the menu tray in mobile environments (since the menu closes
 * immediately on pointerup).
 */
```